subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_dppi.h

```diff
@@ -28,7 +28,7 @@ static inline void hal_radio_nrf_ppi_channels_disable(uint32_t mask)
  * Enable Radio on Event Timer tick:
  * wire the EVENT_TIMER EVENTS_COMPARE[0] event to RADIO TASKS_TXEN/RXEN task.
  */
-#define HAL_RADIO_ENABLE_ON_TICK_PPI 0
+#define HAL_RADIO_ENABLE_ON_TICK_PPI 6
```

Comment:
The commit log says the motivation is "so that DPPI 0-4 available for application's use" -- where is that going to be documented?

More generally, PPI values seem like something that should be moved into DT, since they are boot time hardware configuration values. Another benefit would be that users could override this macro with an overlay as they desire.

Would you agree? If so, that also would solve the documentation problem if there is no existing doc, because the pages in the DT bindings index would be the docs.

Reply:
> The commit log says the motivation is "so that DPPI 0-4 available for application's use" -- where is that going to be documented?

I thought this would document it: https://github.com/zephyrproject-rtos/zephyr/blob/45cdc051ed9e25b7252706ac7a7caf2436084631/modules/hal_nordic/nrfx/nrfx_glue.h#L229

> PPI values seem like something that should be moved into DT

Agree.

(Just thinking aloud, PPI/DPPIs and other peripherals are used by controller at scheduled time periods, they can but are not currently been used outside the Zephyr controller. nRF Connect SD controller based application do use outside the controller's scheduled time periods).